tools/qcdrawer.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2017, IBM.
```

Comment:
can you make sure all files you add have copyright 2018?

Reply:
Yes, I will do it.

Reply:
done